.travis.yml

```diff
@@ -23,17 +23,20 @@ install:
   - conda env create -n my_env -f environment.yml
   - source activate my_env
   - conda install cromwell
-  - python3 -m pip install pytest-workflow
+  - python3 -m pip install pytest-workflow pysam
 
 script:
-  - python3 -m pytest --tag $TEST tests/
+  - travis_wait python3 -m pytest --tag $TEST tests/
```

Comment:
Yes this is a real issue. I will update pytest-workflow so we do not have to use travis_wait anymore.